DEVELOPMENT.md

```diff
@@ -17,13 +17,68 @@ GO_BINARY_PATH=go1.11.X
 ```
 
 # Run the Proxy
+
+We provide two ways to run the proxy on your local machine:
+
+1. Using [Docker](https://www.docker.com/) and [Docker Compose](https://docs.docker.com/compose/) (_we suggest this one if you're getting started_)
+2. Natively on your host
+
+.See below for insrtuctions to do both!
+
+## Using Docker
+
+As we said above, we suggest that you use this approach because it simulates a more realistic Athens deployment. This technique does the following, completely inside containers:
+
+1. Builds Athens from scratch
+2. Starts up [MongoDB](https://www.mongodb.com/) and [Jaeger](https://www.jaegertracing.io/)
+3. Configures Athens to use MongoDB for its storage and Jaeger for its distributed tracing
+4. Runs Athens
+
+You'll need [Docker](https://www.docker.com/) and [Docker Compose](https://docs.docker.com/compose/) installed. Once you do, run the below command to set everything up:
+
+```console
+$ make dev-docker
```

Comment:
Just a small typo here, should be `make run-docker`. I think it used to be `dev-docker` right? Looks like the teardown is right though.

Reply:
good catch, thanks @chriscoffee! Change coming